spec.md

```diff
@@ -1460,6 +1460,30 @@ There is a performance penalty for applications that swap memory to disk often.
 - If `memswap_limit` is unset, and `memory` is set, the container can use as much swap as the `memory` setting, if the host container has swap memory configured. For instance, if `memory`="300m" and `memswap_limit` is not set, the container can use 600m in total of memory and swap.
 - If `memswap_limit` is explicitly set to -1, the container is allowed to use unlimited swap, up to the amount available on the host system.
 
+## models
```

Comment:
I think we should put an explicit statement that a model artifact will be pulled and available to use via the inference endpoint APIs **_before_** any dependent service starts.